src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -1346,11 +1346,26 @@ public Task<(long Index, object Result)> DeleteOngoingTask(long taskId, string t
 
         public Task<(long Index, object Result)> ToggleTaskState(long taskId, string taskName, OngoingTaskType type, bool disable, string dbName)
         {
-            var disableEnableCommand =
-                type == OngoingTaskType.Subscription ?
-                    (CommandBase)new ToggleSubscriptionStateCommand(taskName, disable, dbName) :
-                    new ToggleTaskStateCommand(taskId, type, disable, dbName);
-
+            CommandBase disableEnableCommand;
+            switch (type)
+            {
+                case OngoingTaskType.Subscription:
+                    if (taskName == null)
+                    {
+                        if (_server.ServerStore.DatabasesLandlord.DatabasesCache.TryGetValue(dbName, out var databaseTask) == false)
+                            throw new RachisApplyException($"Value cannot be null or empty. Param: {nameof(dbName)}");
```

Comment:
Huh? That exception is _really_ misleading. The `dbName` isn't empty, it is just not a loaded database for this server.